### ISSUE TITLE
Fix Event Windows

### ISSUE THREAD

**Issue by Greatexperiment** [deliciousmods/1956_beta#223]

**Comment by Thinking-waffle** [comment]
Will we keep every icons small
otherwise we could put big icons for everybody but is that really worth it?
I think that you will agree that we have other stuffs to do

**Comment by Greatexperiment** [comment]
Yep - the downside is that we'll lose the TWAS events.

**Comment by SpicyAlfredo** [comment]
yeah but it's a trade off we have to make

**Comment by Thinking-waffle** [comment]
The war at sea events are not the problem per se are they? I am convinced that versions of those with a normal interface are totaly doable.

**Comment by Greatexperiment** [comment]
No, but they changed the interface for the events - which was silly. Someone would need to go back and rewrite the event text and also resize all the images.
